content/en/faq/security/check-policy.md

```diff
@@ -7,8 +7,8 @@ The [`istioctl`](/docs/reference/commands/istioctl) command provides an option f
 
 {{< text bash >}}
 $ istioctl authn tls-check $CLIENT_POD httpbin.default.svc.cluster.local
-HOST:PORT                                  STATUS     SERVER     CLIENT     AUTHN POLICY        DESTINATION RULE
-httpbin.default.svc.cluster.local:8000     OK         mTLS       mTLS       /default            istio-system/default
+HOST:PORT                                                       STATUS     SERVER     CLIENT           AUTHN POLICY     DESTINATION RULE
```

Comment:
Why is there so much space in front of the STATUS field?

Reply:
Fixed. It was some artifact on my terminal screen.